src/Interfaz/PantallaJuego.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Interfaz;
+
+import java.awt.Graphics;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+/**
+ *
+ * @author kevv87
+ */
+public class PantallaJuego {
+    
+    public PantallaJuego(){
+    
+        MarcoJuego pantalla = new MarcoJuego();
+        pantalla.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        pantalla.setVisible(true);
+    
+    }
+    
+}
+
```

Comment:
comentar clases

Reply:
Está comentado we

Reply:
suave q creo q empecé de achrás pa lante xD, le estoy agarrando el toque a la vara